node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -4702,11 +4688,6 @@ mod tests {
             }
         }
     }
-    //
-    // fn node_by_ip_mut(db: &mut NeighborhoodDatabase, ip_addr: IpAddr) -> &mut NodeRecord {
```

Comment:
Seems unneeded after all. Please go and delete it for sure.

Reply:
It's red code, right? I think it's already deleted.

Reply:
Oh... silly me.